Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
> native-Rust

Since you only see this in descriptions, I think the standard is just 'native'. See: [postgres](https://crates.io/crates/postgres)

> Originally written by MongoDB interns.

I don't think this is really relevant information anymore, and it might lead people away. I understand the need to note that it started as an in-house project but is still unofficially supported. Perhaps we can play around with this?

"A native MongoDB driver. Actively maintained, yet unofficially supported for MongoDB 3.0+."